packages/core/core/src/resolveOptions.js

```diff
@@ -50,13 +50,24 @@ export default async function resolveOptions(
   }
 
   let entryRoot = getRootDir(entries);
+  /*
   let projectRootFile =
     (await resolveConfig(
       inputFS,
       path.join(entryRoot, 'index'),
       [...LOCK_FILE_NAMES, '.git', '.hg'],
       path.parse(entryRoot).root,
     )) || path.join(inputCwd, 'index'); // ? Should this just be rootDir
+  */
+
+  let projectRootFile =
```

Comment:
This was necessary in local testing because of https://github.com/parcel-bundler/parcel/issues/7579